src/TodoItem.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.

Reply:
When i did npm run deploy i have Failed to compile.
![1](https://github.com/mate-academy/react_todo-app-with-api/assets/79115881/ba45d762-b767-47bb-b552-684686b427dc)